netty/BUILD.bazel

```diff
@@ -27,3 +27,11 @@ java_library(
         "@io_perfmark_perfmark_api//jar",
     ],
 )
+
+# Not actually shaded, but mirrors the shaded dependencies included in the
+# artifact on Maven Central for usage with maven_install's override_targets.
+java_library(
```

Comment:
Does this only work on Linux?

Reply:
Yes. Epoll is Linux-only, by definition. It doesn't _break_ on other platforms though.